### PR TITLE
CDK Lowcode: allow parametrizing initial page when using PageIncrement

### LINKED ISSUE #20296: CDK Lowcode: allow parametrizing initial page when using PageIncrement
## What
PageIncrement starts counting pages from 0 which is not always a good choice

### PR DESCRIPTION
**Pull request by davydov-d**
## What
PageIncrement starts counting pages from 0 which is not always a good choice 

## How
Allow configuring the number of the start page

Use case: pagination starts from 1 in [Zenloop](https://github.com/airbytehq/airbyte/pull/19624), Delighted(work in progress)

### CODE REVIEW

**Review by girarda — APPROVED** [review]

File: airbyte-cdk/python/setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="airbyte-cdk",
-    version="0.9.5",
+    version="0.9.6",
```
Comment (brianjlai) [inline airbyte-cdk/python/setup.py:18]: since this is a new backwards compatible feature not a bug, we should increment the minor version

**Review by brianjlai — APPROVED** [review]
looks good to me! one note about the minor version increment instead of patch

**Comment by grubberr** [comment]
@davydov-d 
do we wait Monday 28.11 here ?

**Pull request merged**